canonical-web-and-design/maas-ui
# Share header

Done:
- Add a shared package in the monorepo including the Header.
- Use the shared header for both the ui and legacy projects. Part of https://github.com/canonical-web-and-design/maas-ui/issues/280
- Redirect / to /MAAS in the proxy. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/325.
- Link to the angular machine list in the header. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/327.

QA:
- If you have a legacy/.env.local or a ui/.env.development.local then either update or remove them.
- In the root run `./run`.
- Open the site and check that in the header you can navigate between the react/angular clients and that you see your username in the header and can log out.
